examples/ipv6only-app/esp32/idf.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Now that you've removed the reference of idf.sh, can you please remove the file as well?

Reply:
done